tests/SeedTests.elm

```diff
@@ -11,14 +11,19 @@ fixedSeed =
     Random.initialSeed 133742
 
 
+{- This is the expected first int output when running this file with a constant seed.
+
+    (\intFromSeed -> fnvHashString fnvInit "Seed test" |> fnvHash intFromSeed)
+
+-}
 expectedNum : Int
 expectedNum =
-    2800615587
+    -3954212174
```

Comment:
So did I 😄 I was just wondering if you knew the path through the program that seeds are passed before they end up at the hash function. Mostly because I'm curious, and because my attempt at tracing it yesterday was unsuccessful :)